src/pages/call/call.ts

```diff
@@ -2,15 +2,39 @@ import { Component } from '@angular/core';
 import { NavController, NavParams } from 'ionic-angular';
 import { UserData } from '../../providers/user-data';
 
+import { CallingPage } from './calling';
+
 @Component({
   selector: 'page-call',
   templateUrl: 'call.html',
 })
 export class CallPage {
-  constructor(
-      public navCtrl: NavController,
-      public navParams: NavParams,
-      public userData: UserData
-      ) {
-  }
+    phone_number: string = '';
```

Comment:
@adrianq Sure! I like to keep a consistent format, I just didn't realize. By the way, do you use camel case for the functions too?